### ISSUE TITLE
Incomplete documentation in `twisted.logger.Levels` class InvalidLogLevelError

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8490]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)| @acabhishek942 reported|
|-|-|
|Trac ID|trac#8490|
|Type|enhancement|
|Created|2016-06-21 11:05:22Z|

There should be @type documentation for level and @param should be a description, not a type specifier.

<details><summary>Searchable metadata</summary>

```
trac-id__8490 8490
type__enhancement enhancement
reporter__acabhishek942 acabhishek942
priority__normal normal
milestone__New_Logging_System New Logging System
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__logger logger
keywords__easy easy
time__1466507122472402 1466507122472402
changetime__1481166045112251 1481166045112251
version__None None
owner__acabhishek942 acabhishek942

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)|@wsanchez set owner to @wsanchez<br>@wsanchez set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek's avatar" src="https://avatars.githubusercontent.com/u/50864783?s=50" width="50" height="50">](https://github.com/acabhishek)|@acabhishek removed owner<br>@acabhishek set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek's avatar" src="https://avatars.githubusercontent.com/u/50864783?s=50" width="50" height="50">](https://github.com/acabhishek)<a name="note_3"></a>|@acabhishek commented|
|-|-|

 Link to PR - \\https://github.com/twisted/twisted/pull/202

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_4"></a>|@wsanchez set owner to @acabhishek942|
|-|-|

reviewed, thanks for the cleanup

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|GitHub set status to `closed`|
|-|-|

In changeset e6b7b52740663686d7c4faa6305374c11d4efd80
```
#!CommitTicketReference repository="" revision="e6b7b52740663686d7c4faa6305374c11d4efd80"
Merge 8490-acabhishek942-doc: Complete documentation in twisted.logger.Levels class InvalidLogLevelError

Author: acabhishek942
Reviewer: wsanchez
Fixes: #8490
```
